core/src/main/java/org/web3j/tx/Contract.java

```diff
@@ -15,12 +15,7 @@
 import java.io.IOException;
 import java.lang.reflect.Constructor;
 import java.math.BigInteger;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Please avoid wildcard  import, solve this and I will merge it

Reply:
wildcard import has been solved, have a lovely day and thanks a lot